src/Sqs/SqsClient.php

```diff
@@ -124,14 +190,47 @@ private function validateMd5()
                     ->then(
                         function ($result) use ($c, $r) {
                             foreach ((array) $result['Messages'] as $msg) {
+                                $bodyMd5 = self::calculateBodyMd5($msg);
```

Comment:
It would be great if we are checking `MD5OfMessageAttributes` and `MD5OfBody` in the same pattern here.
I'd prefer the "non-nesting if" we have for `MD5OfBody` currently:
``` php
if (isset($msg['MD5OfBody'])
    && self::calculateBodyMd5($msg) !== $msg['MD5OfBody']) ...

// Then
if (isset($msg['MD5OfMessageAttributes'])
    && self:: calculateMessageAttributesMd5($msg) !== $msg['MD5OfMessageAttributes']) ...
```